bay/images/docker/settings.php

```diff
@@ -171,3 +171,13 @@
     $settings['container_yamls'][] = $settings_path . '/envs/' . getenv('LAGOON_ENVIRONMENT_TYPE') . '/services.yml';
   }
 }
+
+// Include branch specific settings and services files.
+if (getenv('LAGOON_GIT_BRANCH')) {
```

Comment:
```suggestion
if (getenv('LAGOON_GIT_SAFE_BRANCH')) {
```

I think `LAGOON_GIT_SAFE_BRANCH` is safer to use, this does some sanitisation on the branch - for example: `feature/sdpsup-3447` would become `feature-sdpsup-3447`. If we're using this for a directory this would be safer as `/` is a directory separator.